src/components/common/molecules/Banner.tsx

```diff
@@ -1,34 +1,114 @@
-import React from 'react';
-import { StyleSheet, View } from 'react-native';
-import Colors from '../../../stylesheets/colors';
+import { MaterialCommunityIcons } from '@expo/vector-icons';
+import React, { useState } from 'react';
+import {
+  StyleProp,
+  StyleSheet,
+  TextStyle,
+  TouchableOpacity,
+  View,
+  ViewStyle,
+} from 'react-native';
 import Typography from '../atoms/Typography';
+import * as Linking from 'expo-linking';
 
 const styles = StyleSheet.create({
   bannerContainer: {
-    backgroundColor: Colors.PURPLE_LIGHT,
+    backgroundColor: 'white',
+    display: 'flex',
+    flexDirection: 'row',
```

Comment:
If you are changing the colors you could check if this is necessary as flex-direction: row is the default direction